safety/scan/main.py

```diff
@@ -196,8 +203,43 @@ def save_report_as(scan_type: ScanType, export_type: ScanExport, at: Path, repor
     with open(at, 'w+') as report_file:
         report_file.write(report)
 
+def build_meta(target: Path) -> Dict[str, Any]:
+    """
+    Build the meta JSON object for a file.
+
+    Args:
+        target (Path): The path of the repository.
 
-def process_files(paths: Dict[str, Set[Path]], config: Optional[ConfigModel] = None) -> Generator[Tuple[Path, InspectableFile], None, None]:
+    Returns:
+        Dict[str, Any]: The metadata dictionary.
+    """
+    target_obj = target.resolve()
+    git_utils = GIT(target_obj)
+
+    git_data = git_utils.build_git_data()
+    git_metadata = {
+        "git_branch": git_data.branch if git_data else None,
```

Comment:
You can drop the git prefix in the field names, they are now self-describing. So branch instead of git_branch.